sites/public/src/components/listing/ListingMap.tsx

```diff
@@ -0,0 +1,41 @@
+import React from "react"
+import { GoogleMap, Marker, useJsApiLoader } from "@react-google-maps/api"
+import { Listing } from "@bloom-housing/backend-core/types"
+
+type ListingMapProps = {
+  listing: Listing
+  googleMapsHref: string
+}
+
+const containerStyle = {
+  display: "block",
+  width: "100%",
+  height: "400px",
+}
+
+const ListingMap = (props: ListingMapProps) => {
+  const { isLoaded } = useJsApiLoader({
+    googleMapsApiKey: process.env.googleMapsApiKey,
+  })
+
+  const listing = props.listing
+  const latitudeLongitude = {
+    lat: listing.buildingAddress.latitude,
+    lng: listing.buildingAddress.longitude,
+  }
+  const marker = <Marker position={latitudeLongitude}></Marker>
+
+  return isLoaded ? (
+    <a href={props.googleMapsHref}>
+      <div className="listing-map">
```

Comment:
do we need this div?

Reply:
Thanks for pointing it out, we don't need it. This was just used previously to set the min height https://github.com/bloom-housing/ui-components/blob/main/src/page_components/listing/ListingMap.scss